pkg/certificates/ca_test.go

```diff
@@ -518,7 +517,14 @@ sa6wJArEJueVGTZbXePe5zaBUWAHdPX7DYeVISHO4WkOvEquwT1BwJ/v8KO77aKv
 2HcWw8NEqjF9Enb+ieyBD9Ds04NFFAaPDHA/8eWL+PXuMLtN2ALbomG1UcfsrnEL
 qI0vTc9/cZNK/A3W8rUFjat6jPUW1CsZm8yvJ3ORU01V4xVfZMRH3JzwB0WqRGVo
 FYeY7rWtiZX43Wq4szS4xWy7nlk0Jg==
------END PRIVATE KEY-----`)
+-----END PRIVATE KEY-----`
```

Comment:
```suggestion
-----END PRIVATE KEY-----` // NOSONAR 
```
To make sonarqube happy, we will refactor this at some point so these are generated at test time, but for now...

Reply:
Dang, it did not work :( 

I am not sure in which line the comment needs to be added, maybe in the first one instead?

Reply:
I think it just needed a `make fmt`. it seems happier now.

Reply:
We are still getting:
![image](https://github.com/user-attachments/assets/f575a041-1952-4623-b9b4-dd4f4045743b)
:(

Reply:
Yup. trying something else now. Trying to wrap it in a function. That worked.